back/src/main/resources/properties/datasource.yml

```
@@ -11,14 +11,11 @@ spring:
         # schema: rouby
-  jdbc:
-    batch:
-      size: 10000
   sql:
     init:
-      mode: never
-      # schema-locations: classpath:db/schema.sql
-      # data-locations: classpath:db/alter.sql # db 시작할때 실행시킬 script
+      mode: always
+      schema-locations: classpath:db/schema.sql
+      #data-locations: classpath:db/data.sql
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)